apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -901,7 +924,7 @@ lookup_from_all_nodes(ChainName, AuthenticatorID) ->
                 node_error => HelpFun(maps:map(Fun, ErrorMap), reason)
             }};
         {error, ErrL} ->
-            {400, #{
+            {500, #{
```

Comment:
Just confirming: is there a test case that covers this?

Reply:
I'll add one.